lintro/ai/review/progress.py

```
@@ -0,0 +1,185 @@
+        if self._progress is None or self._task_id is None:
+            return
+        self._progress.update(self._task_id, advance=1)
+
+    def on_complete(self, *, total_findings: int) -> None:
+        """Stop the bar and print the completion summary."""
+        if self._progress is not None:
+            self._progress.stop()
+            self._progress = None
+        noun = "finding" if total_findings == 1 else "findings"
+        self._console.print(
+            f"[bold green]✓[/bold green] Review complete — " f"{total_findings} {noun}",
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)